recipes-multimedia/gstreamer/files/0013-yocto-drop-opus-raw-audio-data-according-to-clip-meta-data.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/ext/opus/gstopusdec.c b/ext/opus/gstopusdec.c
```

Comment:
Please add patch header which would have atleast the details of previous Reference